model/client/admin/data/download_shard_request.go

```diff
@@ -14,3 +14,7 @@ type ShardDescriptor struct {
 func (*DownloadShardRequest) Method() string {
 	return "download_shard"
 }
+
+func (*DownloadShardRequest) Validate() error {
```

Comment:
should we assert on shards not being nil? Bit stupid if user does send empty tho